boilerplate/openshift/golang-osd-operator/build-opm-catalog.sh

```diff
@@ -71,7 +71,7 @@ function check_bundle_contents_cmd() {
 # Check we are running an opm supported container engine
 function check_opm_supported_container_engine() {
     local image_builder=${1}
-    if [[ "$image_builder" != "docker" && "$image_builder" != "podman" ]]; then
+    if [[ "$image_builder" != docker* && "$image_builder" != "podman" ]]; then
```

Comment:
I wonder if it would be better to have a separate `$CONTAINER_ENGINE_FLAGS` var rather than glomming the args into `$CONTAINER_ENGINE`. But this is a reasonable quick fix.

Reply:
Ideally the plan is to move DVO towards using `build-push` like other OSD operators. The issue is that currently https://github.com/app-sre/deployment-validation-operator distributes an OSD operator, an app-sre operator only deployed to app-sre clusters, and a community operator all of which overlap in some way in terms of build artifacts. Given the need to fix this before 4.12 a quick solution is preferred in the interim. Thanks.